src/SprykerSdk/Zed/Benchmark/Business/PhpBench/PhpBenchRunner.php

```diff
@@ -0,0 +1,176 @@
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\PhpBenchConfigurationTransfer $phpBenchConfigurationTransfer
+     *
+     * @return int
+     */
+    protected function runTestsForAllDirectories(PhpBenchConfigurationTransfer $phpBenchConfigurationTransfer): int
+    {
+        $testDirectories = $this->findDirectoriesUnderPath($this->config->getTestsFolder());
+        foreach ($testDirectories as $testDirectoryInformation) {
+            $commandExitCode = $this->runCommand(
+                $testDirectoryInformation->getRealPath(),
+                $phpBenchConfigurationTransfer->getIterations(),
+                $phpBenchConfigurationTransfer->getRevolutions()
+            );
+
+            if ($commandExitCode !== static::EXIT_CODE_SUCCESS) {
+                return $commandExitCode;
+            }
+        }
+
+        return static::EXIT_CODE_SUCCESS;
+    }
+
+    /**
+     * @param string $path
+     * @param int $iterations
+     * @param int $revolutions
+     *
+     * @throws \Symfony\Component\Process\Exception\ProcessFailedException
+     *
+     * @return int
+     */
+    protected function runCommand(string $path, int $iterations, int $revolutions): int
+    {
+        $bootstrapFilePath = $this->getPathToBootstrap($path);
+
+        $command = 'php vendor/bin/phpbench run %s --bootstrap=%s --report=aggregate';
+        $command = sprintf($command, $path, $bootstrapFilePath);
+
+        if ($iterations) {
+            $command .= ' --iterations=' . $iterations;
+        }
+
+        if ($revolutions) {
+            $command .= ' --revs=' . $revolutions;
+        }
+
+        $process = $this->createProcess($command);
+        $process->run();
+
+        if (!$process->isSuccessful()) {
+            throw new ProcessFailedException($process);
+        }
+
+        echo $process->getOutput();
+
+        return (int)$process->getExitCode();
+    }
+
+    /**
+     * @param string $path
+     *
+     * @throws \SprykerSdk\Zed\Benchmark\Business\Exception\InvalidBootstrapException
+     *
+     * @return string
+     */
+    protected function getPathToBootstrap(string $path): string
+    {
+        $filePath = sprintf('%s' . DIRECTORY_SEPARATOR . '%s', $path, 'bootstrap.php');
+        if (file_exists($filePath)) {
+            return $filePath;
+        }
+        $defaultBootstrapFile = $this->findDefaultBootstrapFile($path);
+
+        if (!$defaultBootstrapFile) {
+            throw new InvalidBootstrapException('Unable to find bootstrap file. Please add one or check if tests folder name contains tests layer name.');
+        }
+
+        return $defaultBootstrapFile;
+    }
+
+    /**
+     * @param string $path
+     *
+     * @return string|null
+     */
+    protected function findDefaultBootstrapFile(string $path): ?string
+    {
+        $defaultBootstrapFolders = $this->findDirectoriesUnderPath(realpath(__DIR__ . '/../../../../../../bootstrap'));
```

Comment:
This `../../..` needs explanation to me.

Reply:
It's a module root. I'll move it to constant or smth like that